README.md

```diff
@@ -148,7 +176,8 @@ When reconnecting to an existing chat session, the bot will send a message conta
 | `WebChat.show()`                         | Show the chat widget, send initPayload if the chat is in open state and not initialized                                                                                  |
 | `WebChat.hide()`                         | Hide the chat widget                                                                                                                                                     |
 | `WebChat.isVisible()`                    | Get the shown/hidden state of the widget                                                                                                                                 |
-| `WebChat.send(payload, text: optionnal)` | send a payload (`/hello` to Push. If `text` is specified, it will be displayed as a user message. If not specified, no user message will be displayed |                                                                       |
+| `WebChat.send(payload, text: optionnal)` | send a payload (`/hello` to Push. If `text` is specified, it will be displayed as a user message. If not specified, no user message will be displayed |
+| `WebChat.reload()` | Reload the chat widget |
```

Comment:
What means reload the chat widget? clean all the messages, generating a new user Urn? remember that generate new userUrn would duplicate contacts